### PR TITLE
Typography [WEB-1697]

### PR DESCRIPTION
**Pull request by johnkim-det**
Update Typography module to match new requirements. 

Large/Default/Small sizes for Title/Body/Label, and a Code component with one size.

Using antd Typography components for implementation to include antd's dynamic truncation/tooltip by row functionality.

Using font sizes/line heights from previous version of Typography component. May need adjustment per design review.

### CODE REVIEW

File: src/kit/Typography.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <Typography.Text className={getClassName('label', size)} ellipsis={truncate}>
+      {children}
+    </Typography.Text>
+  );
+};
+
+export const Code: React.FC<Props> = ({ children, truncate }: Omit<Props, 'size'>) => {
+  return (
+    <Typography.Paragraph className={getClassName('code')} ellipsis={truncate}>
+      {' '}
```
Comment (EmilyBonar) [inline src/kit/Typography.tsx:56]: nit: why this extra space?

File: src/kit/Typography.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { EllipsisConfig } from 'antd/es/typography/Base';
+import React from 'react';
+
+import { ValueOf } from 'kit/utils/types';
+
+import css from './Typography.module.scss';
+
+export const TypographySize = {
+  Default: 'default',
+  L: 'large',
```
Comment (EmilyBonar) [inline src/kit/Typography.tsx:11]: nit: the large and default sizes are very difficult to tell apart. i'd like us to make the large larger or the default smaller, maybe both
Screenshot 2023-11-01 at 09-42-20 Standalone Design Kit

File: src/kit/Typography.module.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  /* Title */
+
+  /* font-size */
+  --title-size-l: 24px;
+
+  /* line-height */
+  --title-ln-l: 32px;
+  --title-size-default: 22px;
+  --title-ln-default: 28px;
+  --title-size-s: 18px;
```
Comment (EmilyBonar) [inline src/kit/Typography.module.scss:13]: nit: move the sizes out of the line height section

Comment (EmilyBonar) [reply src/kit/Typography.module.scss:13]: this applies to body and label as well

**Review by EmilyBonar — CHANGES_REQUESTED** [review]
Looking good just a few changes

File: src/kit/Typography.module.scss (hunk 3 above)
Comment (johnkim-det) [reply src/kit/Typography.module.scss]: changing this to just group styles by title/body/label type

File: src/kit/Typography.tsx (hunk 2 above)
Comment (johnkim-det) [reply src/kit/Typography.tsx:11]: decreasing Title Default and Small and increasing Body and Label Large, and tweaking the line heights to be divisible by 4px @michielpr

**Review by EmilyBonar — APPROVED** [review]
LGTM as long as you get the ok from design

**Pull request merged**